build.sbt

```diff
@@ -15,13 +15,16 @@ resolvers := Seq(
   "OJO Snapshots" at s"https://oss.jfrog.org/oss-snapshot-local"
 ) ++ resolvers.value
 
-val eventuateVersion = "0.10"
+val eventuateVersion = "0.10-SNAPSHOT"
```

Comment:
This should be reverted before merge

Reply:
Hmm ... we cannot really _revert_ this, right? The build will fail on a `0.10`. I think we could either leave it as it is (unreleased master build of eventuate-chaos depends on unreleased eventuate) or we switch to `0.11-M1` which I want to release soon.

Reply:
Of course we should not leave it as it is. I just switched master of eventuate to `0.11-SNAPSHOT`. (should have happened after the 0.10 release ...). So we could use `0.11-SNAPSHOT` here. But actually I would prefer `0.11-M1` now, which I just released ...